src/main/java/com/facebook/presto/maven/ServiceDescriptorGenerator.java

```diff
@@ -119,7 +119,7 @@ private URLClassLoader createClassloaderFromCompileTimeDependencies()
         List<Class<?>> implementations = Lists.newArrayList();
         List<String> classes = FileUtils.getFileNames(classesDirectory, "**/*.class", null, false);
         for (String classPath : classes) {
-            String className = classPath.substring(0, classPath.length() - 6).replace('/', '.');
+            String className = classPath.substring(0, classPath.length() - 6).replace('/', '.').replace('\\', '.');
```

Comment:
Wouldn't it make it more sense, instead of adding another replace to change the existing one into

`.replace(File.separatorChar, '.')`

which would work on any operating system?

Reply:
Look my pull request #11